src/utils/utils.ts

```diff
@@ -98,9 +99,17 @@ export const transformRequestObject = (
     max_age,
     code_challenge,
     code_challenge_method,
+    login_hint,
   } as AuthRequest;
 };
 
+export const stripLoginHintFromQueryParams = (
+  authRequest: AuthRequest
+): AuthRequest => {
+  authRequest.login_hint = undefined;
```

Comment:
I puzzled over whether it would be better to create a new object with login_hint removed rather than mutate the existing object but concluded that in this case it makes no practical difference :)